### PR TITLE
RHMAP-1471 UPS integration documentation - 2.3.1

### PR DESCRIPTION
**Pull request by rsmeral**
* renamed push starter template apps, because they contained the phrase "Simple Push" which was misleading 
* microcopy corrections

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -152,7 +152,7 @@
         "id": "pushstarter_project",
         "priority": 1,
         "name": "Push Notification Hello World",
-        "description": "An example project using the Unified Push Service built-into the platform",
+        "description": "An example project using the push server integrated in the Platform",
```
Comment (matzew) [inline global.json]: we could say it's the integrated UnifiedPush Server, no ?

**Comment by matzew** [comment]
:+1:

File: global.json (hunk 1 above)
Comment (rsmeral) [inline global.json]: I discussed this with some people (IIRC @balunasj and possibly others) and my understanding is, we are still not completely sure how to market the integrated push server - do we expose the fact it's AG UPS? Do we try to hide it?
Do we keep the brand *UnifiedPush Server* all throughout the docs and UI, or do we just say it's "push functionality within RHMAP"? Is the server "included", "integrated", ...? 
I think I'll leave the current vague formulation until we're really clear on what the message is.

**Comment by rsmeral** [comment]
@jasonmadigan It seems I don't have write permission here, so I can't merge. Would you?

**Comment by jasonmadigan** [comment]
@rsmeral done

**Comment by jasonmadigan** [comment]
fhcap bumped

**Pull request merged**